src/main/java/com/palantir/gradle/gitversion/NativeGitDescribe.java

```diff
@@ -51,7 +51,7 @@ private String runGitCmd(String... commands) throws IOException, InterruptedExce
         cmdInput.add("git");
         cmdInput.addAll(Arrays.asList(commands));
         ProcessBuilder pb = new ProcessBuilder(cmdInput);
-        pb.directory(directory);
+        pb.directory(directory.getParentFile());
```

Comment:
why is this necessary?

Reply:
Here `directory` refers to the `.git` directory. In a worktree `.git` is a file (containing the path to the actual `.git` directory). Adding `.getParentFile()` gets us up to the repository root, which is always an actual directory and just as good of a directory to execute git commands from.

Reply:
I like this. Maybe now is a good time to rename or add documentation around this `directory` variable.